pages/_document.tsx

```diff
@@ -40,6 +40,14 @@ class MyDocument extends Document {
       <Html lang="en-US">
         <Head>
           <meta name="description" content="" />
+          <meta content="#fff" name="theme-color" />
```

Comment:
Next.js warn me to not use these meta tags on `_document` let's move these lines to SEO component. Could you do that?